.github/workflows/autoupdate.yml

```
@@ -0,0 +1,68 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 */3 * * *"
+  workflow_dispatch:
+
+
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        sudo pip3 install setuptools wheel
+        sudo pip3 install yq
+        pip3 install -r scripts/requirements.txt
+    - name: Run autoupdate
+      run: |
+        git config user.email "actions@github.com"
+        git config user.name "Github Actions"
+        git fetch origin
+        
+        for file in manifests/*
+        do
+          git reset --hard master
```

Should this be origin/master?

```suggestion
          git reset --hard origin/master
```

---

Same thing - the `cron` trigger always checks out the default branch, and there's no possibility something is commited to the local master.